contracts/lib/OrderValidator.sol

```diff
@@ -369,7 +374,8 @@ contract OrderValidator is Executor, ZoneInteraction {
                 orderParameters.offer = extendedOffer;
             }
 
-            // Loop through offer and ensure at least as much on returned offer
+            // Loop through each offer and ensure at least as much on returned
```

Comment:
🤔 
This sounds weird.

Reply:
It was written by 0 and I was scared to materially change it, so I just tried to make it a wee bit clearer 😅

Reply:
"Loop through each new offer and ensure the new amounts are at least as much as the respective original amounts"?

Reply:
Sounds good.  Updated